wfl/Models/FoodOrderStatusDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Symphono.Wfl.Models
+{
+    public class FoodOrderStatusDto
+    {
+        public string Status { get; set; }
```

Comment:
I think that's fine, as long as we discussed and thought about it.  I also would NOT take in an integer.  You can pass the enum as a string in a DTO object.  Thanks